great_tables/_utils_render_html.py

```diff
@@ -225,7 +236,7 @@ def create_columns_component_h(data: GTData) -> str:
                     colspan=len(stub_layout),
                     style=_flatten_styles(styles_stubhead),
                     scope="colgroup" if len(stub_layout) > 1 else "col",
-                    id=_process_text_id(stub_label),
+                    id=_create_element_id(table_id, stub_label),
```

Comment:
Just noting for the future (nothing needs to be done here) but the stub doesn't really have an ID value (I suppose the column that created the stub labels could be it?) and it often won't have a label unless `tab_stubhead()` was called.

The issue with all this is that for the stubhead `<th>` element we get `id=""`, which is invalid HTML according to https://validator.w3.org/nu/.